snorkel/labeling/model/label_model.py

```diff
@@ -807,25 +781,31 @@ def _break_col_permutation_symmetry(self) -> None:
         mu = self.mu.cpu().detach().numpy()
         P = self.P.cpu().detach().numpy()
         d, k = mu.shape
-
-        # Iterate through the possible perumation matrices and track heuristic scores
-        Zs = []
-        scores = []
-        for idxs in permutations(range(k)):
-            Z = np.eye(k)[:, idxs]
-            Zs.append(Z)
-
-            # If Z and P commute, get heuristic score, else skip
-            if np.allclose(Z @ P, P @ Z):
-                scores.append(self._count_accurate_lfs(mu @ Z))
-            else:
-                scores.append(-1)
-
-        # Set mu according to highest-scoring permutation
+        probs_sum = sum([mu[i : i + k] for i in range(0, self.m * k, k)]) @ P
+
+        m = Munkres()
```

Comment:
No need for such a short var name here. It's used far away from this definition and just the once.